tests/testAutopas/C08TraversalTest.cpp

```diff
@@ -0,0 +1,102 @@
+/**
+ * C08TraversalTest.cpp
+ *
+ *  Created on: 5/24/18
+ *     Aauthor: F. Gratl
+ */
+
+
+#ifdef AUTOPASAUTOPAS_OPENMP
```

Comment:
AUTOPASAUTOPAS_OPENMP
maybe we should rename the macro, so that this problem does not occur anymore (through replacing)
e.g. AUTOPASOPENMP
would make it more unreadable however...

Reply:
Hmm, I would keep it for now and see if this happens more often since `AUTOPASOPENMP` would be a real mess.

Reply:
fixed cedd624ff74d3a56950ac24e83d6558ef3b718a1